### ISSUE TITLE
Use a property to enable OpenTelemetry for JDBC 

### ISSUE THREAD

**Issue by brunobat** [quarkusio/quarkus#30532]
### Description

This is a consistency issue, we need to align the Quarkus OpenTelemetry configuration with: https://github.com/quarkusio/quarkus/issues/8710

We need properties to enable and disable the jdbc tracing.
Example:
```

quarkus.datasource.postgres.jdbc.tracing=true
quarkus.datasource.postgres.jdbc.tracing.enabled=true
```

**Comment by michalvavrik** [comment]
This looks interesting to me, but I can't look into it before the end of February for lack of spare time. I'll look then whether the issue has been implemented or look into it.

**Comment by brunobat** [comment]
Sure @michalvavrik. Thanks very much for your help :)

**Comment by gsmet** [comment]
@brunobat you agree that we will never have both OpenTracing and OpenTelemetry around and so we can use the exact same properties?

**Comment by gsmet** [comment]
@michalvavrik assigning to you. If at some point you're not able to work on that, please ping me. Thanks!

**Comment by gsmet** [comment]
Pull request for OpenTracing is here: https://github.com/quarkusio/quarkus/pull/26144 .

**Comment by michalvavrik** [comment]
thx @gsmet , I'll be certainly able to work on that during late February, np

**Comment by brunobat** [comment]
Yes @gsmet. OpenTracing and OpenTelemetry extensions are incompatible and we already validate that they must not be used together and fail the build.
